[2022-01-31 00:00:09,317] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.pre_process_image scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2022-01-31 00:00:09,349] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.pre_process_image scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2022-01-31 00:00:09,360] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 00:00:09,361] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-01-31 00:00:09,362] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 00:00:09,410] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): pre_process_image> on 2022-01-30 00:00:00+00:00
[2022-01-31 00:00:09,419] {standard_task_runner.py:52} INFO - Started process 5533 to run task
[2022-01-31 00:00:09,431] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'pre_process_image', 'scheduled__2022-01-30T00:00:00+00:00', '--job-id', '3121', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpxoqe0fj6', '--error-file', '/tmp/tmp4ehx5hr9']
[2022-01-31 00:00:09,433] {standard_task_runner.py:77} INFO - Job 3121: Subtask pre_process_image
[2022-01-31 00:00:09,565] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.pre_process_image scheduled__2022-01-30T00:00:00+00:00 [running]> on host 218df5ddc363
[2022-01-31 00:00:09,706] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=pre_process_image
AIRFLOW_CTX_EXECUTION_DATE=2022-01-30T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-30T00:00:00+00:00
[2022-01-31 00:00:09,711] {python.py:152} INFO - Done. Returned value was: None
[2022-01-31 00:00:09,730] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=pre_process_image, execution_date=20220130T000000, start_date=20220131T000009, end_date=20220131T000009
[2022-01-31 00:00:09,816] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-31 00:00:09,924] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
